miden-crypto/src/merkle/smt/large/batch_ops.rs

```diff
@@ -644,6 +645,7 @@ impl<S: SmtStorage> LargeSmt<S> {
             entry_count_delta,
         );
         self.storage.apply(updates)?;
+        self.in_memory_nodes[1] = new_root;
```

Comment:
Similar comment to the one above - though, maybe it is more difficult to do it here.